spec/smoke_test/job_seekers_can_view_homepage_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'browser_test_helper'
+require 'pry'
+
+RSpec.describe 'Page availability', js: true do
+  context 'Job seeker visits home page' do
+    it 'Page should be available' do
```

Comment:
With `rspec` I think it's important to ensure that when you back the test within the `it` block that it reads well. For example, 'it should ensure users can search and view a job vacancy page' would be:

```
it 'should ensure users can search and view a job vacancy page' do
```

Reply:
Yes it reads much better, resolved this in cd60e72